src/main/java/com/openshift/jenkins/plugins/pipeline/dsl/OpenShiftBuilder.java

```diff
@@ -91,8 +93,27 @@ public String getWaitTime(Map<String,String> overrides) {
 			return val;
 		return Long.toString(GlobalConfig.getBuildWait());
 	}
-	
-	
+
+	public String getWaitUnits() { return this.waitUnit; }
+
+    @Override
+    public String getWaitTimeMilli(Map<String, String> overrides) {
```

Comment:
You should be able to move this to IOpenShiftBuilder as a default method, and then share the implementation between the dsl and classic freestyle versions